src/Illuminate/Cookie/Middleware/EncryptCookies.php

```diff
@@ -216,10 +216,9 @@ public function isDisabled($name)
     /**
      * Determine if the cookie contents should be serialized.
      *
-     * @param  string  $name
      * @return bool
      */
-    public static function serialized($name)
+    public static function serialized()
```

Comment:
Removing this is a breaking change for anyone overwriting this method.

Reply:
thank you for your review, 
so we just need a test for cookie serialization?

Reply:
No, I'm saying if you remove this people's apps that override this method will break. Just leave this be.